hlx_statics/blocks/site-wide-banner/site-wide-banner.js

```diff
@@ -0,0 +1,115 @@
+import {
+  createTag,
+  decorateButtons,
+} from "../../scripts/lib-adobeio.js";
+import {
+  fetchSiteWideBanner,
+  getMetadata,
+} from "../../scripts/lib-helix.js";
+import { getVariant } from "../inlinealert/inlinealert.js";
+
+export default async function decorate(block) {
+  const siteParent = block.parentElement.parentElement;
+  const mainArea = document.querySelector('[style*="grid-area: main"]');
+  const sidenavArea = document.querySelector('[style*="grid-area: sidenav"]');
+  const isDocTemplate = getMetadata("template") === "documentation";
+  const productName = getMetadata('product');
+  const isMobile = window.innerWidth < 1025;
+  const paddingTargets = isDocTemplate ? [mainArea, sidenavArea] : [siteParent.nextElementSibling?.nextElementSibling];
+  block.setAttribute('daa-lh', 'site-wide-banner');
+  const allowedColors = ["warning", "success", "info", "neutral", "notice", "light"];
+
+  let banner;
+  try {
+    const bannerSource = isDocTemplate
+      ? await fetchSiteWideBanner()
+      : await fetch(`https://main--adobe-io-website--adobe.hlx.page/franklin_assets/${productName ? `${productName}-` : ''}site-wide-banner.json`).then(resp => {

```

Comment:
This is pointing to the wrong url. It should be:
`https://main--adp-devsite-stage--adobedocs.aem.page/` when on stage/local dev
`https://main--adp-devsite--adobedocs.aem.live/` for prod.